de.philippun1.turtle.json

```diff
@@ -0,0 +1,85 @@
+{
+  "app-id": "de.philippun1.turtle",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "command": "turtle_cli",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--filesystem=home",
+    "--device=dri"
+  ],
+  "cleanup": [
+    "/bin/git2",
+    "/include",
+    "/lib/pkgconfig"
+  ],
+  "modules": [
+    {
+      "name": "turtle",
```

Comment:
this module should be last. We build the dependencies first. (it doesn't matter for run time checked things like python, but it's much easier to read)

Reply:
Don't know why this is not detected as outdated, but it is last now.